### ISSUE TITLE
Not calling SetTabIcon when using On<Android>().SetToolbarPlacement(ToolbarPlacement.Bottom);

### ISSUE THREAD

**Issue by arvindrajachourasiya** [xamarin/Xamarin.Forms#3397]
### Description
I am working on Xamarin.Forms project. In _Shared Project_ when using code below

`On<Xamarin.Forms.PlatformConfiguration.Android>().SetToolbarPlacement(ToolbarPlacement.Bottom);`

Then `TabbedPageRenderer`  class overridden  method` SetTabIcon` not getting called at all in Customrenderer. I need this method to be called to set cutom_layout on Tabs.

MyTabbed.cs //In shared project

```
public class MyTabbedPage : Xamarin.Forms.TabbedPage
    {
        public MyTabbedPage()
        {
            On<Xamarin.Forms.PlatformConfiguration.Android>().SetToolbarPlacement(ToolbarPlacement.Bottom);
        }
    }
```

MyTabbedPageRenderer.cs //In android project
```
public class MyTabbedPageRenderer : TabbedPageRenderer
{
	protected override void SetTabIcon(TabLayout.Tab tab, FileImageSource icon)
	{
		base.SetTabIcon(tab, icon);
		tab.SetCustomView(Resource.Layout.Custom_tab_layou);
		var imageview = tab.CustomView.FindViewById<ImageView>(Resource.Id.icon);
		var tv = tab.CustomView.FindViewById<TextView>(Resource.Id.tv);
		tv.SetText(tab.Text, TextView.BufferType.Normal);          
		imageview.SetBackgroundDrawable(tab.Icon);
	}
}
```

### Steps to Reproduce

1.  Create Xamarin.forms project.
2.  Create 2 files as I have shown above
3. Created custopm renderer for Tabs & override SetTabIcon method

### Expected Behavior

SetTabIcon()  should call

### Actual Behavior

SetTabIcon() not calling

### Basic Information

Microsoft Visual Studio Enterprise 2017 
Version 15.7.5
VisualStudio.15.Release/15.7.5+27703.2042
Microsoft .NET Framework
Version 4.7.03056

Installed Version: Enterprise

Architecture Diagrams and Analysis Tools   00369-60000-00001-AA519
Microsoft Architecture Diagrams and Analysis Tools

Application Insights Tools for Visual Studio Package   8.12.10405.1
Application Insights Tools for Visual Studio

C# Tools   2.8.3-beta6-63029-08. Commit Hash: e9a3a6c0ba5b1fde8b1fff964bdfb3fb768ee2eb
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Visual Studio Tools for Containers   1.1
Develop, run, validate your ASP.NET Core applications in the target environment. F5 your application directly into a container with debugging, or CTRL + F5 to edit & refresh your app without having to rebuild the container.

Mono Debugging for Visual Studio   4.10.5-pre (ab58725)
Support for debugging Mono processes with Visual Studio.

NuGet Package Manager   4.6.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio

Visual Studio Tools for Universal Windows Apps   15.0.27703.2042
The Visual Studio Tools for Universal Windows apps allow you to build a single universal app experience that can reach every device running Windows 10: phone, tablet, PC, and more. It includes the Microsoft Windows 10 Software Development Kit.

Xamarin   4.10.10.2 (35a01d8dc)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   4.12.1 (f3257e429)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin.Android SDK   8.3.3.2 (HEAD/dffc59120)
Xamarin.Android Reference Assemblies and MSBuild support.

Xamarin.iOS and Xamarin.Mac SDK   11.12.0.4 (64fece5)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.

**Comment by arvindrajachourasiya** [comment]
Is there anyone who can help me out?

**Comment by jassmith** [comment]
Basically what GalaxiaGuy said. This is being converted into an enhancement request.

**Comment by jsmarcus** [comment]
Will this be solved by #3967?

**Comment by jfversluis** [comment]
Thanks for this suggestion! As Xamarin.Forms is now in maintenance mode, this will not happen anymore for Xamarin.Forms. We're only adding bugfixes and stability fixes.

If this is still important to you, make sure to check the [.NET MAUI repo](https://github.com/dotnet/maui) and see if it's already on the roadmap. If not, feel free to open a discussion to discuss a change first or open an issue with a detailed feature request. Thanks!
